h/accounts/views.py

```diff
@@ -130,8 +135,60 @@ class AsyncAuthController(AuthController):
 @view_config(attr='forgot_password', route_name='forgot_password')
 @view_config(attr='reset_password', route_name='reset_password')
 class ForgotPasswordController(horus.views.ForgotPasswordController):
-    pass
+    def forgot_password(self):
+        req = self.request
+        schema = req.registry.getUtility(IForgotPasswordSchema)
+        schema = schema().bind(request=req)
+
+        form = req.registry.getUtility(IForgotPasswordForm)
+        form = form(schema)
+
+        if req.method == 'GET':
+            if req.user:
+                return HTTPFound(location=self.forgot_password_redirect_view)
+            else:
+                return {'form': form.render()}
 
+        controls = req.POST.items()
+        try:
+            captured = form.validate(controls)
+        except deform.ValidationFailure as e:
+            return {'form': e.render(), 'errors': e.error.children}
+
+        user = self.User.get_by_email(req, captured['email'])
+        activation = self.Activation()
+        self.db.add(activation)
+        user.activation = activation
+        Str = self.Str
```

Comment:
PEP8 doesn't like this. I know it's comes from horus. Let's make it nicer, though. It's not used in enough places to bother with a local variable. Just use `self.Str` everywhere.